lib/App/Sqitch/Engine/pg.pm

```diff
@@ -137,14 +137,11 @@ has dbh => (
                 connected => sub {
                     my $dbh = shift;
                     $dbh->do('SET client_min_messages = WARNING');
-                    try {
-                        $dbh->do(
-                            'SET search_path = ?',
-                            undef, $self->registry
-                        );
-                        # https://www.nntp.perl.org/group/perl.dbi.dev/2013/11/msg7622.html
-                        $dbh->set_err(undef, undef) if $dbh->err;
-                    };
+                    $dbh->do(
+                        'SET search_path = ?',
```

Comment:
I think in postgresql it is not an error to set the search_path to reference a nonexistent schema.

Reply:
Yes, right, that's fine — better even, because then the query of the `changes` table will fail:

```
david=# create table changes (id int);
CREATE TABLE

david=# set search_path to 'xxx';
SET

david=# insert into foo values(1);
ERROR:  relation "foo" does not exist
LINE 1: insert into foo values(1);
                    ^
```

This is just what we want.

Reply:
Added a comment in c6ccc306 about this.